doc/tutorials/mediasdk-tutorials-cmd-reference.md

```diff
@@ -87,6 +90,16 @@ Tutorial initializes Intel Media SDK session of a specified type. If *-auto* is
 | -hw | Loads HW Media SDK Library implementation |
 | -auto | Automatically choses Media SDK library implementation |
 
+## simple_7_codec
+**simple_7_codec *[-sw | -hw | -auto]***
+Tutorial Showcases MSDK features via call Query functions. The tutorial can be run with the *-hw*, *-sw*, *-auto* or no options, it will report the capability for each codec. Since this tutorial check all supported coded, it will configure a set of the video parameters for each codec by the predefined filling function. Each filling function will try the maximum resolutions, user might change the resolution based on his platform.
```

Comment:
Since this tutorial check -> Since this tutorial checks

Reply:
done